jib-maven-plugin/CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ All notable changes to this project will be documented in this file.
 - Better feedback for build failures ([#197](https://github.com/google/jib/pull/197))
 - Warns if specified `mainClass` is not a valid Java class ([#206](https://github.com/google/jib/issues/206))
 - Warns if build may not be reproducible ([#245](https://github.com/GoogleContainerTools/jib/pull/245))
+- `jibBuildDocker` gradle task to build straight to Docker daemon ([#265](https://github.com/GoogleContainerTools/jib/pull/265))
```

Comment:
Oh I meant move this to `jib-gradle-plugin/CHANGELOG.md`.

Reply:
Oh, should it go in both? i.e. base changelog mentions both, and then each plugin changelog mentions the respective task/goal? Or just the plugin changelogs

Reply:
Wait I see what's happening, nevermind I know what to do